src/cirrus/lib2/eventdb.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import os
+from datetime import datetime
+from enum import Enum, unique
+from typing import Any, Dict, Optional
+
+import boto3
+
+logger = logging.getLogger(__name__)
+
+
+@unique
+class StateEnum(Enum):
+    PROCESSING = "PROCESSING"
+    COMPLETED = "COMPLETED"
+    FAILED = "FAILED"
+    INVALID = "INVALID"
+    ABORTED = "ABORTED"
+
+
+class EventDB:
+    def __init__(self, event_db_and_table_names: Optional[str] = None):
+        self.tsw_client = boto3.client("timestream-write")
+        self.tsq_client = boto3.client("timestream-query")
+
+        if event_db_and_table_names is None:
+            event_db_and_table_names = os.getenv("CIRRUS_EVENT_DB_AND_TABLE")
```

Comment:
It seems like this needs to be added to the default `cirrus.yml` file. Look at `src/cirrus/core/config/default.yml`.